<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addsourcedatapage</class>
 <widget class="QWidget" name="addsourcedatapage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>232</width>
    <height>265</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	background-color:rgb(85, 170, 255)
}
QLabel
{
	font: 12pt &quot;微软雅黑&quot;;
}
QPushButton
{
	background-color: rgba(255, 255, 255, 170);
font: 12pt &quot;微软雅黑&quot;;
	height:40px;
border:none;
border-radius:5px;
}
QLineEdit
{
	background-color: rgba(255, 255, 255, 170);
	font: 12pt &quot;微软雅黑&quot;;
	height:40px;
border:none;
border-radius:5px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_2"/>
     </item>
     <item row="2" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_4"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>权限编号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>姓名</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>电话号码</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>添加</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>密码</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1" colspan="2">
      <widget class="QDateTimeEdit" name="dateTimeEdit"/>
     </item>
     <item row="6" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_8"/>
     </item>
     <item row="7" column="2">
      <widget class="QPushButton" name="pushButton_2">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>部门编号</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>创建时间</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>邮箱</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_5"/>
     </item>
     <item row="4" column="1" colspan="2">
      <widget class="QLineEdit" name="lineEdit_6"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
